packit_service/worker/helpers/testing_farm.py

```diff
@@ -455,14 +480,19 @@ def run_testing_farm(
                 success=True,
                 details={"msg": "Project not allowed to use internal TF."},
             )
-
         self.report_status_to_test_for_test_target(
             state=BaseCommitStatus.running,
             description=f"{'Build succeeded. ' if not self.skip_build else ''}"
             f"Submitting the tests ...",
             target=target,
         )
 
+        compose = self.distro2compose(target)
+
+        if not compose:
+            msg = "We were not able to map distro to TF compose."
+            return TaskResults(success=False, details={"msg": msg})
```

Comment:
Would it be possible to let the user know about it?

E.g. `The compose XY is not available in the {internal/public} Testing Farm infrastructure.`

Ideally, use markdown content for more details:
* List available composes or link the endpoint.
* Suggest checking the config.
* Suggest contacting the team if the valid compose is not available and I, as a user, think it should be there.

Reply:
Updated, I haven't mentioned the mapping because pointing to the mapping code can get outdated and I wasn't able to figure out a clear message describing it (and since people will probably contact us following the message, we can resolve this with them individually maybe?).

Reply:
I agree linking the code is not a good idea. The list might be long but that's ok if it is in the markdown detail.